src/setup.erl

```diff
@@ -858,7 +871,7 @@ run_setup(Parent, Args) ->
     end.
 
 run_setup_(Parent, _Args) ->
-    Res = rpc:multicall(?MODULE, verify_directories, []),
+    Res = maybe_verify_directories(),
```

Comment:
It was the only spot where `multicall` was used. So just to be consistent I guess it is okay to remove it and think about whether a multi-node enforement of anything by setup makes sense as an option.